package.json

```diff
@@ -6,7 +6,7 @@
   "addon": {
     "$ABOUT": "use these variables fill the moustache templates",
     "id": "tracking-protection-messaging@shield-study.mozilla.com",
-    "name": "Button Icon Preference Study (Shield Study Example)",
+    "name": "Tracking Protection Messaging Shield Study",
```

Comment:
Now that this has a more accurate name, the `description` field below (L16) feels redundant and less than informative (it's now the same value as `name`).

Reply:
My [earlier PR](https://github.com/biancadanforth/tracking-protection-shield-study/pull/53) updates the description field to say: "The purpose of this study is to test various types of messaging around Tracking Protection in Firefox."